docs/configuration.md

```diff
@@ -2306,6 +2306,50 @@ showDF(properties, numRows = 200, truncate = FALSE)
   </tr>
 </table>
 
+### Spark Application Garbage Collector
+
+<table class="table">
+  <tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
+  <tr>
+    <td><code>spark.cleaner.periodicGC.interval</code></td>
+    <td>30min</td>
+    <td>
+      Controls how often to trigger a garbage collection.
+    </td>
+  </tr>
+  <tr>
+    <td><code>spark.cleaner.referenceTracking</code></td>
+    <td>true</td>
+    <td>
+      Controls whether a ContextCleaner should be created when a SparkContext initializes.
```

Comment:
I'd just say it controls whether context cleaning is enabled at all. This sort of suggests it controls whether it's created at a certain time as opposed to another time.

Reply:
Rephrased.